src/components/Card.js

```diff
@@ -0,0 +1,128 @@
+// modules
+import { useState, useEffect } from "react";
+
+const Card = ({ addListItems, burgerProduct, handleModalOpen }) => {
+	// state
+	const [counter, setCounter] = useState(0);
+	const [subtotal, setSubtotal] = useState(0);
+
+	// destructure burger
+	const { name, imgUrl, price } = burgerProduct;
+
+	// When counter is changed - update subtotal
+	useEffect(() => {
+		setSubtotal(price * counter);
+	}, [counter]);
+
+	const handleAdd = (e) => {
+		setCounter(counter + 1);
+		handleUnclick(e);
+	};
+
+	const handleSubtract = (e) => {
+		if (counter > 0) {
+			setCounter(counter - 1);
+		}
+		handleUnclick(e);
+	};
+	const handleSubmit = (e) => {
+		e.preventDefault();
+
+		// check to see if user selected counter
+		if (counter > 0) {
+			const burgerObj = {
+				name: name,
+				price: price,
+				counter: counter,
+				subtotal: subtotal,
+			};
+			addListItems(burgerObj);
+		} else {
+			alert("Please enter an amount");
+		}
+	};
+
+	// remove focus effect after click
+	const handleUnclick = (e) => {
+		e.target.blur();
+	};
+
+	return (
+		<form
+			onSubmit={(e) => {
+				handleSubmit(e);
+			}}
+			action="submit"
+			className="burger-card"
+		>
+			<div className="img-container">
+				<label htmlFor="open-modal" className="sr-only">
+					Click to customize the burger
```

Comment:
Clear message to support screen readers 🙌🏻

Reply:
🙌🏾